### PR TITLE
Matched a couple random functions 

### PR DESCRIPTION
**Pull request by kklions1**

### CODE REVIEW

File: src/EA8AE0.c [hunk 1]
```diff
@@ -44,7 +45,28 @@ INCLUDE_ASM(s32, "EA8AE0", func_8024140C_EA9EEC);
 
 #include "world/common/AddPlayerHandsOffset.inc.c"
 
-INCLUDE_ASM(s32, "EA8AE0", func_802417AC_EAA28C);
+// INCLUDE_ASM(s32, "EA8AE0", func_802417AC_EAA28C);
+
+
+extern s32 D_80243DD8_EAC8B8;
+s32 D_80243DDC_EAC8BC = 0;
```
Comment (ethteck) [inline src/EA8AE0.c:52]: As discussed in discord, please remove this line setting this var to 0 and wrap the function in NON_MATCHING

File: src/effects/rising_bubble.c [hunk 2]
```diff
@@ -8,7 +8,43 @@ INCLUDE_ASM(s32, "effects/rising_bubble", rising_bubble_main);
 void rising_bubble_init(void) {
 }
 
-INCLUDE_ASM(s32, "effects/rising_bubble", rising_bubble_update);
+void rising_bubble_update(EffectInstance* effect) {
+    RisingBubbleFXData* risingBubbleFXData = effect->data.risingBubble;
```
Comment (ethteck) [inline src/effects/rising_bubble.c:12]: We name the data vars here just `data`

File: src/effects/rising_bubble.c [hunk 3]
```diff
@@ -8,7 +8,43 @@ INCLUDE_ASM(s32, "effects/rising_bubble", rising_bubble_main);
...
+    s32 temp_a2 = risingBubbleFXData->unk_18;
+    s32 temp_v0;
```
Comment (ethteck) [inline src/effects/rising_bubble.c:14]: These 3 temps can be removed since they're unused

File: src/effects/rising_bubble.c [hunk 4]
```diff
@@ -8,7 +8,43 @@ INCLUDE_ASM(s32, "effects/rising_bubble", rising_bubble_main);
...
+        risingBubbleFXData->unk_20++;
+        if (risingBubbleFXData->unk_20 >= 0x20) {
+            risingBubbleFXData->unk_20 -= 0x20;
+        }
+        
+        risingBubbleFXData->unk_10 += 0.1;
+        return;
+    }
+    
+    risingBubbleFXData->unk_14 = 0xFF;
```
Comment (ethteck) [inline src/effects/rising_bubble.c:42]: Please write all the hex numbers here in decimal. We only use hex for enums, pretty much, and decimal values are used for positions, measurements, color, etc

File: src/world/area_trd/trd_06/9B1AA0.c [hunk 5]
```diff
@@ -10,7 +10,12 @@ INCLUDE_ASM(s32, "world/area_trd/trd_06/9B1AA0", func_80240854_9B1FE4);
 
 INCLUDE_ASM(s32, "world/area_trd/trd_06/9B1AA0", func_8024093C_9B20CC);
 
-INCLUDE_ASM(s32, "world/area_trd/trd_06/9B1AA0", func_80240A5C_9B21EC);
+extern s32 D_802451F0;
```
Comment (ethteck) [inline src/world/area_trd/trd_06/9B1AA0.c:13]: Please put all externs at the top of the file before the first function

**Review by ethteck — CHANGES_REQUESTED** [review]
Nice job :) just some small things

**Comment by BowserSlug** [comment]
🚀🚀 This PR matches 2 functions (+0.01%) on `us`.

File: src/effects/rising_bubble.c [hunk 6]
```diff
@@ -8,7 +8,40 @@ INCLUDE_ASM(s32, "effects/rising_bubble", rising_bubble_main);
...
+        
+        if (data->unk_18 < 0) {
+            shim_remove_effect(effect); 
+            return;
+        }
+        
+        if (temp_a2 >= 6) { 
+            data->unk_14 += (120 - data->unk_14) * 0.3;
+        } else {
+            data->unk_14 = (temp_a2 * 20);
```
Comment (ethteck) [inline src/effects/rising_bubble.c:27]: no parens needed around this expression

File: src/EA8AE0.c [hunk 7]
```diff
@@ -44,7 +47,26 @@ INCLUDE_ASM(s32, "EA8AE0", func_8024140C_EA9EEC);
 
 #include "world/common/AddPlayerHandsOffset.inc.c"
 
+#ifdef NON_MATCHING 
+ApiStatus func_802417AC_EAA28C(Evt* script, s32 isInitialCall) {
+    Bytecode* temp = script->ptrReadPos; 
+    
+    if (isInitialCall != 0) {
```
Comment (ethteck) [inline src/EA8AE0.c:54]:
```suggestion
    if (isInitialCall) {
```

File: src/EA8AE0.c [hunk 8]
```diff
@@ -44,7 +47,26 @@ INCLUDE_ASM(s32, "EA8AE0", func_8024140C_EA9EEC);
 
 #include "world/common/AddPlayerHandsOffset.inc.c"
 
+#ifdef NON_MATCHING 
+ApiStatus func_802417AC_EAA28C(Evt* script, s32 isInitialCall) {
+    Bytecode* temp = script->ptrReadPos;
```
Comment (ethteck) [inline src/EA8AE0.c:52]: we name these temps `args`

File: src/EA8AE0.c [hunk 9]
```diff
@@ -44,7 +47,26 @@ INCLUDE_ASM(s32, "EA8AE0", func_8024140C_EA9EEC);
...
+ApiStatus func_802417AC_EAA28C(Evt* script, s32 isInitialCall) {
+    Bytecode* temp = script->ptrReadPos; 
+    
+    if (isInitialCall != 0) {
+        D_80243DD8_EAC8B8 = 0;
+    }
+    if (D_80243DD8_EAC8B8 != 0) {
+        D_80243DD8_EAC8B8 = 0;
+
+        dead_evt_set_variable(script, *temp, D_80243DDC_EAC8BC);
```
Comment (ethteck) [inline src/EA8AE0.c:60]:
```suggestion
        dead_evt_set_variable(script, *args++, D_80243DDC_EAC8BC);
```

**Review by ethteck — CHANGES_REQUESTED** [review]
almost

File: src/effects/rising_bubble.c [hunk 10]
```diff
@@ -8,7 +8,40 @@ INCLUDE_ASM(s32, "effects/rising_bubble", rising_bubble_main);
...
+        data->unk_10 += 0.1;
+        return;
+    }
+    
+    data->unk_14 = 255;
+    data->unk_1C++;
+    data->unk_08 += 2.0f;
+}
+
+
```
Comment (nanaian) [inline src/effects/rising_bubble.c:44]: Remove this double empty line

**Review by nanaian — CHANGES_REQUESTED** [review]

**Pull request merged**